<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Neuron"/>
    <addaction name="actionNew_Layer"/>
    <addaction name="actionNew_Net"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Neuron">
   <property name="text">
    <string>New Neuron</string>
   </property>
  </action>
  <action name="actionNew_Layer">
   <property name="text">
    <string>New Layer</string>
   </property>
  </action>
  <action name="actionNew_Net">
   <property name="text">
    <string>New Net</string>
   </property>
  </action>
  <action name="actionDelete_Neuron">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionDelete_Layer">
   <property name="text">
    <string>Delete Layer</string>
   </property>
  </action>
  <action name="actionDelete_Net">
   <property name="text">
    <string>Delete Net</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
